{"uid":"993c5f33872308ab","name":"Search for existing project test","fullName":"tests.workspaces_test.TestWorkspaces#test_search_project","historyId":"89e36300939846c9ff65d6b54eed8329","time":{"start":1717377486485,"stop":1717377486485,"duration":0},"description":"Search for an existing project","descriptionHtml":"<p>Search for an existing project</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564499f06e8a <unknown>\n#1 0x564499bf045c <unknown>\n#2 0x564499c3c5b5 <unknown>\n#3 0x564499c3c671 <unknown>\n#4 0x564499c80f14 <unknown>\n#5 0x564499c5f4dd <unknown>\n#6 0x564499c7e2cc <unknown>\n#7 0x564499c5f253 <unknown>\n#8 0x564499c2f1c7 <unknown>\n#9 0x564499c2fb3e <unknown>\n#10 0x564499ecd2cb <unknown>\n#11 0x564499ed1377 <unknown>\n#12 0x564499eb9dfe <unknown>\n#13 0x564499ed1e42 <unknown>\n#14 0x564499e9e79f <unknown>\n#15 0x564499ef6178 <unknown>\n#16 0x564499ef634b <unknown>\n#17 0x564499f05fbc <unknown>\n#18 0x7fa182294ac3 <unknown>","statusTrace":"self = <tests.workspaces_test.TestWorkspaces object at 0x7f73519176b0>\n\n    @pytest.fixture(autouse=True)\n    def setup_method_fixture(self):\n>       login(self.about_page, self.login_page)\n\ntests/workspaces_test.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/workspaces_test.py:15: in login\n    about_page.click_login_link()\npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"ef0e5306be50adc924e93c3bce0194cc\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f73519a6d40>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x564499f06e8a <unknown>\nE       #1 0x564499bf045c <unknown>\nE       #2 0x564499c3c5b5 <unknown>\nE       #3 0x564499c3c671 <unknown>\nE       #4 0x564499c80f14 <unknown>\nE       #5 0x564499c5f4dd <unknown>\nE       #6 0x564499c7e2cc <unknown>\nE       #7 0x564499c5f253 <unknown>\nE       #8 0x564499c2f1c7 <unknown>\nE       #9 0x564499c2fb3e <unknown>\nE       #10 0x564499ecd2cb <unknown>\nE       #11 0x564499ed1377 <unknown>\nE       #12 0x564499eb9dfe <unknown>\nE       #13 0x564499ed1e42 <unknown>\nE       #14 0x564499e9e79f <unknown>\nE       #15 0x564499ef6178 <unknown>\nE       #16 0x564499ef634b <unknown>\nE       #17 0x564499f05fbc <unknown>\nE       #18 0x7fa182294ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setup_method_fixture","time":{"start":1717377488129,"stop":1717377498498,"duration":10369},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564499f06e8a <unknown>\n#1 0x564499bf045c <unknown>\n#2 0x564499c3c5b5 <unknown>\n#3 0x564499c3c671 <unknown>\n#4 0x564499c80f14 <unknown>\n#5 0x564499c5f4dd <unknown>\n#6 0x564499c7e2cc <unknown>\n#7 0x564499c5f253 <unknown>\n#8 0x564499c2f1c7 <unknown>\n#9 0x564499c2fb3e <unknown>\n#10 0x564499ecd2cb <unknown>\n#11 0x564499ed1377 <unknown>\n#12 0x564499eb9dfe <unknown>\n#13 0x564499ed1e42 <unknown>\n#14 0x564499e9e79f <unknown>\n#15 0x564499ef6178 <unknown>\n#16 0x564499ef634b <unknown>\n#17 0x564499f05fbc <unknown>\n#18 0x7fa182294ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 884, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/workspaces_test.py\", line 25, in setup_method_fixture\n    login(self.about_page, self.login_page)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/workspaces_test.py\", line 15, in login\n    about_page.click_login_link()\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Click Login link","time":{"start":1717377488129,"stop":1717377498498,"duration":10369},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564499f06e8a <unknown>\n#1 0x564499bf045c <unknown>\n#2 0x564499c3c5b5 <unknown>\n#3 0x564499c3c671 <unknown>\n#4 0x564499c80f14 <unknown>\n#5 0x564499c5f4dd <unknown>\n#6 0x564499c7e2cc <unknown>\n#7 0x564499c5f253 <unknown>\n#8 0x564499c2f1c7 <unknown>\n#9 0x564499c2fb3e <unknown>\n#10 0x564499ecd2cb <unknown>\n#11 0x564499ed1377 <unknown>\n#12 0x564499eb9dfe <unknown>\n#13 0x564499ed1e42 <unknown>\n#14 0x564499e9e79f <unknown>\n#15 0x564499ef6178 <unknown>\n#16 0x564499ef634b <unknown>\n#17 0x564499f05fbc <unknown>\n#18 0x7fa182294ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Search for an existing project","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564499f06e8a <unknown>\n#1 0x564499bf045c <unknown>\n#2 0x564499c3c5b5 <unknown>\n#3 0x564499c3c671 <unknown>\n#4 0x564499c80f14 <unknown>\n#5 0x564499c5f4dd <unknown>\n#6 0x564499c7e2cc <unknown>\n#7 0x564499c5f253 <unknown>\n#8 0x564499c2f1c7 <unknown>\n#9 0x564499c2fb3e <unknown>\n#10 0x564499ecd2cb <unknown>\n#11 0x564499ed1377 <unknown>\n#12 0x564499eb9dfe <unknown>\n#13 0x564499ed1e42 <unknown>\n#14 0x564499e9e79f <unknown>\n#15 0x564499ef6178 <unknown>\n#16 0x564499ef634b <unknown>\n#17 0x564499f05fbc <unknown>\n#18 0x7fa182294ac3 <unknown>","statusTrace":"self = <tests.workspaces_test.TestWorkspaces object at 0x7f73519176b0>\n\n    @pytest.fixture(autouse=True)\n    def setup_method_fixture(self):\n>       login(self.about_page, self.login_page)\n\ntests/workspaces_test.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/workspaces_test.py:15: in login\n    about_page.click_login_link()\npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"ef0e5306be50adc924e93c3bce0194cc\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f73519a6d40>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x564499f06e8a <unknown>\nE       #1 0x564499bf045c <unknown>\nE       #2 0x564499c3c5b5 <unknown>\nE       #3 0x564499c3c671 <unknown>\nE       #4 0x564499c80f14 <unknown>\nE       #5 0x564499c5f4dd <unknown>\nE       #6 0x564499c7e2cc <unknown>\nE       #7 0x564499c5f253 <unknown>\nE       #8 0x564499c2f1c7 <unknown>\nE       #9 0x564499c2fb3e <unknown>\nE       #10 0x564499ecd2cb <unknown>\nE       #11 0x564499ed1377 <unknown>\nE       #12 0x564499eb9dfe <unknown>\nE       #13 0x564499ed1e42 <unknown>\nE       #14 0x564499e9e79f <unknown>\nE       #15 0x564499ef6178 <unknown>\nE       #16 0x564499ef634b <unknown>\nE       #17 0x564499f05fbc <unknown>\nE       #18 0x7fa182294ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","steps":[{"name":"public ip address","time":{"start":1717377498511,"stop":1717377498527,"duration":16},"status":"passed","steps":[],"attachments":[{"uid":"6184829a609602a9","name":"HTTP logs of http://checkip.amazonaws.com/","source":"6184829a609602a9.txt","type":"text/plain","size":359}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"e69a950c57cf442","name":"Cookies","source":"e69a950c57cf442.json","type":"application/json","size":3634},{"uid":"1f43e2624c52e42","name":"Session Storage","source":"1f43e2624c52e42.json","type":"application/json","size":359},{"uid":"93d166037c6af04c","name":"Local Storage","source":"93d166037c6af04c.json","type":"application/json","size":294},{"uid":"4eec330a2d74cdbd","name":"Console Logs","source":"4eec330a2d74cdbd.json","type":"application/json","size":6729},{"uid":"cc5db33f4d99ea26","name":"Full Page Screenshot","source":"cc5db33f4d99ea26.png","type":"image/png","size":5696115},{"uid":"d7c0c691f8a1745","name":"URL","source":"d7c0c691f8a1745.uri","type":"text/uri-list","size":23},{"uid":"f5fad1afbfe2f188","name":"Network Logs","source":"f5fad1afbfe2f188.json","type":"application/json","size":941499}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"epic","value":"Workspaces"},{"name":"story","value":"WorkSpaces Creation and Editing Functionality"},{"name":"severity","value":"normal"},{"name":"tag","value":"run(order=6)"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"workspaces_test"},{"name":"subSuite","value":"TestWorkspaces"},{"name":"host","value":"fv-az1114-714"},{"name":"thread","value":"2346-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.workspaces_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=6)"]},"source":"993c5f33872308ab.json","parameterValues":[]}